pkg/sql/pg_catalog.go

```diff
@@ -23,8 +23,8 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/keys"
 	"github.com/cockroachdb/cockroach/pkg/security"
+	resolver2 "github.com/cockroachdb/cockroach/pkg/sql/catalog/resolver"
```

Comment:
did you intend to keep "resolver2" here? If yes then maybe give it a better name.

Reply:
Done.